compiler/nim.nim

```diff
@@ -33,6 +33,9 @@ when defined(profiler) or defined(memProfiler):
   {.hint: "Profiling support is turned on!".}
   import nimprof
 
+when defined(libbacktrace):
```

Comment:
IMO every new `defined` symbol in nim sources should be prefixed with nim => nimlibbacktrace; this prevents clashes with user defined symbols (but what'd be even better is supporting namespace, eg `defined(nim.libbacktrace)`)